src/groups/circle_group.jl

```diff
@@ -15,7 +15,6 @@ CircleGroup() = GroupManifold(Circle{ℂ}(), MultiplicationOperation())
     else
         return merge_traits(
             IsGroupManifold(M.op),
-            HasBiinvariantMetric(),
```

Comment:
So it‘s invariant but not bi-invariant? Should we maybe have a trait for the single-invariant metric?

Reply:
Eh, I just haven't fully undone the removal of this trait. I think we need to redo invariant metrics anyway. Note that it only appears (on master branch) in the *non*-metric path of `active_traits`, so it was useless anyway.

Reply:
Ok, we can surely redo that later (maybe when moving MatricManifold to ManifoldsBase, as I once tried to start).